doc/meetings/2023_02_17.md

```diff
@@ -0,0 +1,32 @@
+# Meeting 2023-02-17
+
+https://github.com/ElektraInitiative/PermaplanT/issues/37#issuecomment-1433926689
+
+## Questions from Lukas and Markus:
+
+1. Should we pin the rust version in any way, or is it ok to always use the latest version?
+    - we use latest
```

Comment:
```suggestion
    - we use Rust version 1.67.1 (2023-02-09)
```

please also update doc/decisions/backend_programming_language.md and link there.